zio/zio.go

```diff
@@ -87,3 +64,7 @@ func Extension(format string) string {
 		return ""
 	}
 }
+
+type NoCloser struct{ io.Writer }
+
+func (_ *NoCloser) Close() error { return nil }
```

Comment:
Nit: I think this would be a little nicer if exposed as a function rather than a type, like io/ioutil.NopCloser, but we can do that later.
```suggestion
// NoCloser returns an io.WriteCloser with a no-op Close method wrapping w.
func NoCloser(w io.Writer) io.WriteCloser { return &noCloser{w} }

type noCloser struct{ io.Writer }

func (_ *noCloser) Close() error { return nil }
```

Reply:
Thanks, I'll make the change before I merge and follow the go pattern by changing the to NopCloser.